content/rc/administration/configure/backups.md

```diff
@@ -64,42 +63,6 @@ wish to backup.
 ![Identify the redis
 version](/images/rc/backups-version1.png?width=800&height=448)
 
-### For Redis versions prior to 3.2.x
```

Comment:
@Aviad-Abutbul Was I correct to remove this? Are the after 3.2.x instructions below correct for Pro also?

Reply:
Yes